### DISCUSSION TITLE
Named views in SSR

### DISCUSSION THREAD

**Discussion by pcaselas** [posva/unplugin-vue-router#464] [Questions and Help] [answered]
`index@aux.vue` doesn't seem to work in SSR mode.

Is that how it's supposed to be? SSR is not supported yet?

**Comment by posva** [comment, marked as answer]
It shouldn't be related to SSR. If you are using Nuxt, I think they don't support named views with that syntax

**Reply by pcaselas** [reply]
I am not using Nuxt, I am using plain vue 3.

**Reply by posva** [reply]
then yeah, it should work as it's not related to SSR.

**Reply by kas84** [reply]
Not for me, I mean, let's see if I am understanding it correctly, if I have something like /invoices/index@aux.vue, the route will be /invoices/ but I get to search for an aux in my IDE? 

Do I need to set something up in vue router for this or it's supposed to work out of the box? 

That's what I don't like about index.vue because then I get lots of index.vue or [id].vue and navigating to those with the IDE is not great.

**Reply by posva** [reply]
I don't understand... The route is `/invoices` but you get a [named view](https://uvr.esm.is/guide/file-based-routing.html#named-views) it can be consumed by `<RouterView name="aux" />`

About having too many index.vue files #448
